src/sentry/identity/oauth2.py

```diff
@@ -30,24 +31,88 @@ class OAuth2Provider(Provider):
 
     oauth_scopes = ()
 
+    def get_oauth_access_token_url(self):
+        # for most cloud-based providers, take this off of the class
+
+        if self.oauth_access_token_url is not '':
+            return self.oauth_access_token_url
+
+        # # check the model for instalation information
```

Comment:
these commented out blocks will be used later when the installation flow has completed successfully